grails-app/controllers/org/pih/warehouse/api/PutawayApiController.groovy

```diff
@@ -116,6 +118,7 @@ class PutawayApiController {
 
             putawayItem.availableItems =
                     inventoryService.getAvailableBinLocations(putawayItem.currentFacility, putawayItem.product)
+            putawayItem.quantityAvailable = inventoryService.getQuantity(putawayItem.currentFacility.inventory, putawayItem.currentLocation, putawayItem.inventoryItem)
```

Comment:
this can be removed, because the qty is already calculated the create endpoint
I think you can also remove the availableItems above